src/FEValues/interface_values.jl

```diff
@@ -0,0 +1,265 @@
+# Defines InterfaceValues and common methods
+"""
+    InterfaceValues(quad_rule::FaceQuadratureRule, func_interpol::Interpolation, [geom_interpol::Interpolation])
+
+An `InterfaceValues` object facilitates the process of evaluating values, averages, jumps and gradients of shape functions
+and nodal functions on the interfaces of finite elements.
```

Comment:
Probably I should dig here: Why do you differentiate between shape functions and "nodal" functions? What exactly is the latter?

Reply:
I think it's `function_*` stuff, where we have the function value at the nodes

Reply:
You mean e.g. `function_value`?

Reply:
Yes